src/scss/tools/mixins/_typography.scss

```diff
@@ -21,3 +21,13 @@
     @include font-line-ratio($sizeName);
     @include font-rhythm($sizeName);
 }
+
+// step-by-step eliminate this insanity
+// replaces @extend .copy--small;
+// replaces @include font-ratio-rhythm(prim-s);
+@mixin font-ratio-rhythm-prim-s {
```

Comment:
It's not actually used :) so this can 👋

Reply:
1. wanted to take a step-by-step approach and not initially mess with the typography robin and plas setup, but we can discuss at the office: i hate having to use the negative margins this way to achieve "vertical rhythm".
1. wanted to also discuss the best approach, pull in the mixin or paste the result of the mixin. found a few conflicting styles, so i pasted the result in and eliminated the excess styling.